web/src/enterprise/campaigns/global/create/CampaignCLIHelp.tsx

```diff
@@ -92,14 +92,15 @@ export const CampaignCLIHelp: React.FunctionComponent<Props> = ({ className }) =
                         />
                     </pre>
                     <p>
-                        See the examples below for more real-world usecases and{' '}
+                        See the examples below for more real-world usecases and read "{' '}
```

Comment:
```suggestion
                        See the examples below for more real-world use cases and read "{' '}
```

Should be two words 🙂